v1/media.yaml

```diff
@@ -254,11 +245,9 @@ definitions:
         type: string
         description: description of the image from Wikimedia Commons
     required:
-      - titles
-      - file_page
+      - section_id
       - type
-      - thumbnail
-      - license
+      - original
```

Comment:
There is something wrong here @mdholloway : the spec test requires `section_id` and `type` to be present, but the definition additionally requires the `original` field. These need to match.